### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by yurikfm** [petrovvlad/freeLib#125] [General]
привет.
а есть способ собирать freelib на минт19?
стандартная сборка не идет, вылазит ошибка на 31% ...

**Comment by petrovvlad** [comment]
Какая ошибка?

**Comment by yurikfm** [comment]
я собрал в текстовый файл саму ошибку и отправил на почту.

**Reply by petrovvlad** [reply]
Вам надо установить более свежую версию Qt.

**Comment by yurikfm** [comment]
ясно.
